docs/CHANGELOG.md

```diff
@@ -1,8 +1,47 @@
 Antares Changelog
 =================
 
+v8.4.2 (01/2022)
```

Comment:
I can see that there are missing references to the following PRs.

https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/669
https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1032
https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1040
https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1041
https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1050
https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1058

Is it expected ?

Reply:
No, I must have forgotten these. I will add them

Reply:
I won't add #1050 since this compilation error wasn't present in v8.4.1, it would be confusing. As for #1058, it's about SonarCloud and very minor, I won't add it either

Reply:
#669 was already present. Added the rest